go.sum

```diff
@@ -264,6 +264,7 @@ github.com/mattermost/gosaml2 v0.3.2 h1:kq2dY5qUe6fPPHra171GVlgo+ycBsEog0gZMetxL
 github.com/mattermost/gosaml2 v0.3.2/go.mod h1:Z429EIOiEi9kbq6yHoApfzlcXpa6dzRDc6pO+Vy2Ksk=
 github.com/mattermost/ldap v0.0.0-20191128190019-9f62ba4b8d4d h1:2DV7VIlEv6J5R5o6tUcb3ZMKJYeeZuWZL7Rv1m23TgQ=
 github.com/mattermost/ldap v0.0.0-20191128190019-9f62ba4b8d4d/go.mod h1:HLbgMEI5K131jpxGazJ97AxfPDt31osq36YS1oxFQPQ=
+github.com/mattermost/mattermost-server v5.11.1+incompatible h1:LPzKY0+2Tic/ik67qIg6VrydRCgxNXZQXOeaiJ2rMBY=
```

Comment:
please revert this line

Reply:
No prob! Should be fixed in b0dd349b9ef95e07d055c349731ef3444b82b811.

So I avoid this in the future, is there a command I unknowingly issued that update this file?  When do we intentionally update it?  I believe I understand [the files use,](https://golang.org/cmd/go/#hdr-Module_downloading_and_verification) but wanted to try and not make this mistake again.  Thanks!

Reply:
normally you shouldn't have to worry about not adding it, but in this because of some odd dependency, it's pulling in the older mm server version. Will have to investigate why this is happening. if ever in doubt just run `go mod tidy` and that should fix things up.